webpack.config.js

```diff
@@ -26,8 +26,8 @@ module.exports = {
 	},
 	entry: {
 		index: './client/blocks/index.js',
-		payment_requests_customizer:
-			'./client/settings/payment-requests/index.js',
+		payment_requests_settings:
```

Comment:
- Moved the page to the `entrypoints` directory (see pc4etw-jM-p2 for background)
- Ensured the name of the directory matches the name of the entrypoint more closely.